addons/html_editor/static/src/link/link.js

```diff
@@ -0,0 +1,33 @@
+import { Component, useRef, useState } from "@odoo/owl";
+import { useOverlay } from "../core/overlay_plugin";
+
+export class LinkPopover extends Component {
+    static template = "html_editor.linkPopover";
+    static props = {
+        // overlay props
+        dispatch: Function,
+        offset: Function,
+        el: {
+            validate: (el) => el.nodeType === Node.ELEMENT_NODE,
+        },
```

Comment:
you don t use all of this props ?

Reply:
Indeed, I will remove the redundant ones :pray: